File: Day_17/Exception.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
class Solution {

    static void Main(String[] args) {
        string S = Console.ReadLine();
        try {
            int strToInt = Int32.Parse(S);
            Console.WriteLine(strToInt);
        } catch(Exception e) {
            Console.WriteLine("Bad String");
        }
    }
}

File: Day_17/Exception.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String S = in.next();
        try {
            int strToInt = Integer.parseInt(S);
            System.out.println(strToInt);
        } catch(Exception e) {
            System.out.println("Bad String");
        }
    }
}